docs/sources/guides/whats-new-in-v7-1.md

```diff
@@ -16,51 +16,49 @@ This topic includes the release notes for the Grafana v7.1, which is currently i
 - [**Provisioning of apps**]({{< relref "#provisioning-of-apps" >}})
 - [**Azure Monitor Datasource**]({{< relref "#azure-monitor-datasource" >}})
-- [**Influx Datasource**]({{< relref "#influx-datasource" >}})
 - [**Deep linking for Google Cloud Monitoring (formerly named Google Stackdriver) datasource**]({{< relref "#deep-linking-for-google-cloud-monitoring-formerly-named-google-stackdriver-datasource" >}})
-- [**Transforms**]({{< relref "#transforms" >}})
-- [**Stat panel text mode**]({{< relref "#stat-panel-text-mode" >}})
-- [**Unification of Explore modes**]({{< relref "#explore-modes-unified" >}})
-- [**Time range picker update**]({{< relref "#time-range-picker-update" >}})
 - [**Grafana Enterprise features**]({{< relref "#grafana-enterprise-features" >}})
-  - [**Support for HashiCorp Vault**]({{< relref "#support-for-hashicorp-vault" >}})
-  - [**Internal links for Elastic**]({{< relref "#internal-links-for-elastic" >}})
-
-## Query history search
+  - [**Secret management with HashiCorp Vault**]({{< relref "#support-for-hashicorp-vault" >}})
+  - [**Monthly schedules in reports**]({{< relref "#support-for-monthly-schedules-in-reports" >}})
 
-In Grafana v 7.1 we are introducing search functionality in Query history. You can search across queries and your comments. It is especially useful in combination with a time filter and data source filter. Read more about Query history [here]({{<relref "../features/explore/index.md#query-history" >}}).
+## Influx Datasource
 
-{{< docs-imagebox img="/img/docs/v71/query_history_search.gif" max-width="800px" caption="Query history search" >}}
+Support for Flux and Influx v2 has been added. The InfluxData blog post, [How to Build Grafana Dashboards with InfluxDB, Flux and InfluxQL](https://www.influxdata.com/blog/how-grafana-dashboard-influxdb-flux-influxql/) explains the changes in depth.
 
-## Provisioning of apps
+## Query history search
 
-Grafana v7.1 adds support for provisioning of app plugins. This allows app plugins to be configured and enabled/disabled using configuration files. Read more about provisioning of app plugins [here]({{<relref "../administration/provisioning.md#plugins" >}}).
+In Grafana v 7.1 we are introducing search functionality in Query history. You can search across queries and your comments. It is especially useful in combination with a time filter and data source filter. Read more about [Query history here]({{<relref "../features/explore/index.md#query-history" >}}).
 
-## Azure Monitor Datasource
+{{< docs-imagebox img="/img/docs/v71/query_history_search.gif" max-width="800px" caption="Query history search" >}}
 
-Support for multiple dimensions has been added to all services in the Azure Monitor datasource. This means you can now group by more than one dimension with time series queries. With the Kusto based services, Log Analytics and Application Insights Analytics, you can also select multiple metrics as well as multiple dimensions.
+## Explore modes unified
```

Comment:
Explore needs to be updated to reflect this change.

Reply:
@oddlittlebird not sure what you mean?

Reply:
Hmm yeah I don't think we update the general explore docs after this. @kaydelaney can you take a look what needs to be updated there?

Reply:
Oh, do you mean the docs for Explore are not updated yet?

Reply:
Yes, and then linked from this document. I thought I saw this doc edit go through. Maybe it hasn't been merged yet?